ibis/tests/all/test_window.py

```diff
@@ -298,3 +305,44 @@ def test_bounded_preceding_windows(backend, alltypes, df, con, window_fn):
     left, right = result.val, expected.val
 
     backend.assert_series_equal(left, right)
+
+
+@pytest.mark.parametrize(
+    ('result_fn', 'expected_fn'),
+    [
+        param(
+            lambda t, win: t.double_col.mean().over(win),
+            lambda gb: (gb.double_col.transform('mean')),
+            id='mean',
+        ),
+        param(
+            lambda t, win: mean_udf(t.double_col).over(win),
+            lambda gb: (gb.double_col.transform('mean')),
+            id='mean_udf',
+            marks=pytest.mark.udf,
```

Comment:
udf tests are currently skipped in ci?

Reply:
They are enabled now (#2363 enabled them)

https://github.com/ibis-project/ibis/blob/e61dd4c1d6753e4920b2b6a67e6fc382c6d37deb/.github/workflows/main.yml#L162
(PySpark/Spark CI job no longer has `not udf`)